oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1479,6 +1548,65 @@ uint8_t pldm::responder::oem_ibm_platform::Handler::fetchRealSAIStatus()
     return PLDM_SENSOR_NORMAL;
 }
 
+void pldm::responder::oem_ibm_platform::Handler::
```

Comment:
nit : the dbus could have been just one function which can take dbusMapping , 3 functions seems like more code